packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -127,6 +128,7 @@ const php = new WebPHP(undefined, {
 	rewriteRules: wordPressRewriteRules,
 });
 
+const memoizedFetch = createMemoizedFetch(monitoredFetch);
```

Comment:
Should this be called [below](https://github.com/WordPress/wordpress-playground/pull/1306/files#diff-5466414cb25ca570e2095b662e08efbdcb5edd0aee22de36ee9c2c2eb2cdfb50R151) where `monitoredFetch()` is used?

Reply:
Good spot, I messed up conflict resolution on GitHub :D